include/Entity.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef ENTITY_HPP
+# define ENTITY_HPP
+
+# include <string>
+# include <OgreSceneNode.h>
+# include <OgreEntity.h>
+
+# include "Vect.hpp"
+# include "Game.hpp"
+
+class Entity
+{
+  protected:
+    Ogre::Entity *ogreEntity;
+    Ogre::SceneNode *sceneNode;
+
+  public:
+    constexpr Entity(void)
+      : ogreEntity(nullptr)
+      , sceneNode(nullptr)
+    {}
+
+    Entity(std::string const &mesh,
+	   Ogre::SceneNode *parent = Game::getGame().getRenderer().getSceneManager().getRootSceneNode());
+
+    constexpr Entity(Entity const &e)
+      : ogreEntity(e.ogreEntity)
+      , sceneNode(e.sceneNode)
+    {}
+
+    constexpr Entity(Entity &&e)
+      : ogreEntity(std::move(e.ogreEntity))
+      , sceneNode(std::move(e.sceneNode))
+    {
+      e.ogreEntity = nullptr;
+      e.sceneNode = nullptr;
```

Comment:
Since the destructor does not act on the pointers of Entity, why the = nullptr?
(note that using a class after it has moved is not defined after the standart)